src/components/Terminal/XTermDefaultStyle.ts

```diff
@@ -117,12 +117,4 @@ export default `
 	height: 1px;
 	overflow: hidden;
 }
-
-.xterm-dim {
```

Comment:
I don't think this is intended, maybe outdated node modules?

Reply:
yeah looks like it